### DISCUSSION TITLE
DOS attack ?!?

### DISCUSSION THREAD

**Discussion by pcar75** [jamulussoftware/jamulus#3064] [Bug Investigation]
Bonjour, 
I just noticed something strange that seems like a DOS (Denial Of Service) Attack.  
https://explorer.jamulus.io reports 3 servers (seemingly not affiliated) full with undescribed connections
2023-05-11 16:04 (New York time zone)
Genre 2 :
	Slackware Jammer	Amsterdam	Netherlands	10 / 10	9	51.158.144.35	22124	3.9.1	Linux
        Heart_Of_The_Beast	VULTR Chicago	United States	16 / 16	88	45.63.69.94	22124	3.9.1	Linux
Genre Rock :
	NGKnew	Cologne	Germany	20 / 20	16	3.126.153.135	22124	3.7.0	Linux

Is it ? 
Any relation with current bot discussions ?

**Comment by mcfnord** [comment]
Started about 3 days ago. Not connected to my bots.

I don't think these three servers block unexpected UDP traffic. Maybe we should recommend blocking UDP traffic on public servers. Then a directory would know the IP of the origin.

**Comment by rdica** [comment]
> Bonjour, I just noticed something strange that seems like a DOS (Denial Of Service) Attack. https://explorer.jamulus.io reports 3 servers (seemingly not affiliated) full with undescribed connections 2023-05-11 16:04 (New York time zone) Genre 2 : Slackware Jammer Amsterdam Netherlands 10 / 10 9 51.158.144.35 22124 3.9.1 Linux Heart_Of_The_Beast VULTR Chicago United States 16 / 16 88 45.63.69.94 22124 3.9.1 Linux Genre Rock : NGKnew Cologne Germany 20 / 20 16 3.126.153.135 22124 3.7.0 Linux
> 
> Is it ? Any relation with current bot discussions ?

@pcar75 We can discuss this next time we meet, I am in contact with these server operators. This is not related to @mcfnord Ear and lobby bots.
